applications/GeoMechanicsApplication/tests/cpp_tests/custom_processes/test_geo_apply_constant_scalar_value_process.cpp

```diff
@@ -80,4 +80,83 @@ KRATOS_TEST_CASE_IN_SUITE(GeoApplyConstantScalarValueProcess_FinalizeDoesNothing
     }))
 }
 
+KRATOS_TEST_CASE_IN_SUITE(GeoApplyConstantScalarValueProcess_Throws_WhenTryingToFixIntVariable,
+                          KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    Model      current_model;
+    ModelPart& r_model_part = current_model.CreateModelPart("Main", 2);
```

Comment:
```suggestion
    auto& r_model_part = current_model.CreateModelPart("Main", 2);
```
This suggestion also applies to the other new test cases.

Reply:
Done